docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,763 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - 192.168.100.50-192.168.100.65
+
+$ kubectl apply -f metallb-configmap.yaml
+configmap/config created
+
+$ kubectl get all -n metallb-system
+NAME                              READY   STATUS    RESTARTS   AGE
+pod/controller-684f5d9b49-jkzfk   1/1     Running   0          2m14s
+pod/speaker-b457r                 1/1     Running   0          2m14s
+pod/speaker-bzmmj                 1/1     Running   0          2m14s
+pod/speaker-gphh5                 1/1     Running   0          2m14s
+pod/speaker-lktgc                 1/1     Running   0          2m14s
+
+NAME                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                 AGE
+daemonset.apps/speaker   4         4         4       4            4           beta.kubernetes.io/os=linux   2m14s
+
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/controller   1/1     1            1           2m14s
+
+NAME                                    DESIRED   CURRENT   READY   AGE
+replicaset.apps/controller-684f5d9b49   1         1         1       2m14s
+```
+
+
+##### Install Nginx
+
+```bash
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+$ helm repo update
+$ helm install ingress-nginx ingress-nginx/ingress-nginx
+```
+
+
+##### Create ingress routing for accessing application deployed in cluster
+
+```bash
+$ cat cluster-ingress.yaml
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: sample-nginx-ingress-hostrouting
+  namespace: sample-domain1-ns
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+  - host: 'domain1.org'
+    http:
+      paths:
+      - path:
+        backend:
+          serviceName: sample-domain1-cluster-cluster-1
+          servicePort: 8001
+
+$ kubectl apply -f cluster-ingress.yaml
+```
+
+##### Create ingress for admin console access
+
+
+```bash
+$ cat admin-ingress.yaml
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: sample-nginx-ingress-adminconsole
+  namespace: sample-domain1-ns
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  rules:
+  - host:
+    http:
+      paths:
+      - path:
+        backend:
+          serviceName: sample-domain1-admin-server
+          servicePort: 7001
+
+$ kubectl apply -f admin-ingress.yaml
+
+```
+
+##### Verify ingresses are running
+
+```bash
+$ kubectl get ingresses -n sample-domain1-ns
+NAME                               CLASS    HOSTS         ADDRESS          PORTS   AGE
+sample-nginx-ingress-adminconsole   <none>  *                              80      7m18s
+sample-nginx-ingress-hostrouting   <none>   domain1.org   192.168.100.50   80      7m18s
+```
+
+##### Access admin console 
+Access admin console using loadbalancer ip (192.168.100.50)
```

Comment:
Access admin console using loadbalancer ip (192.168.100.50) -> Access the Administration Console using the load balancer IP(`192.168.100.50`)